Configuration.props

```diff
@@ -53,7 +53,7 @@
     <HostCcName Condition=" '$(HostCcName)' == '' ">cc</HostCcName>
     <HostCxxName Condition=" '$(HostCxxName)' == '' ">c++</HostCxxName>
     <MakeConcurrency Condition=" '$(MakeConcurrency)' == '' And '$(HostCpuCount)' != '' ">-j$(HostCpuCount)</MakeConcurrency>
-    <ManagedRuntime Condition=" '$(ManagedRuntime)' == '' And '$(OS)' != 'Windows_NT' ">mono</ManagedRuntime>
+    <ManagedRuntime Condition=" '$(ManagedRuntime)' == '' And '$(OS)' != 'Windows_NT' ">mono --debug</ManagedRuntime>
```

Comment:
this will break the `'$(ManagedRuntime)' == 'mono'` check below :)

Reply:
oh, damn, you're right. Ugh.